src/EssentialsPE/BaseFiles/BaseAPI.php

```diff
@@ -428,9 +428,9 @@ public function getCurrencySymbol(): string{
     /**
      * Return the top 5 players with most money
      *
-     * @param CommandSender $player
+     * @param CommandSender|Player $player
      *
-     * @return bool
+     * @return bool|void
```

Comment:
this looks suspicious, it's typehinted as `bool`.

Reply:
Hmm looking at [this](https://github.com/poggit-orphanage/EssentialsPE/blob/fc3799185f756c27633e41c9b8161726c0f5ec8f/src/EssentialsPE/BaseFiles/BaseAPI.php#L435) its typehinted as `void`.

Reply:
since it's hinted as void, it's impossible for it to return bool.